ramls/schemas/logs/recordLog.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "File that should be exported",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "Record UUID",
+      "type": "string"
+    },
+    "hrid": {
+      "description": "Record HRID",
+      "type": "string"
+    },
+    "title": {
+      "description": "Record title",
+      "type": "string"
+    },
+    "recordType": {
+      "description": "Record type",
+      "type": "object",
+      "$ref": "../profiles/recordTypes.json"
+    },
+    "affectedRecord": {
```

Comment:
looks like something wrong here, error log contains record uuid, hrid, title, and affected record, that contains record uuid, hrid, title. Is it should be so?

Reply:
yes, we have the following user stories MDEXP-257 (Instances), MDEXP-259 (Holdings), MDEXP-260 (Items), where we should return json response contains information about error for a record and their associated records, like holdings record and the associated instance record, I think it's better to implement such way to return all information in the one response